### ISSUE TITLE
getDataIDs failing

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/geoknife#55]
```
XPath error : Undefined namespace prefix
XPath error : Invalid expression
 Show Traceback
 
 Rerun with Debug
 Error in xpathApply.XMLInternalDocument(ndoc, path, fun, ..., namespaces = namespaces,  : 
  error evaluating xpath expression //ns:LiteralData 
```

**Comment by jread-usgs** [comment]
the post in GDP UI client is now: <wps:Execute service="WPS" version="1.0.0" xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd">
<ows:Identifier>gov.usgs.cida.gdp.wps.algorithm.discovery.ListOpendapGrids</ows:Identifier>
<wps:DataInputs>
<wps:Input>
<ows:Identifier>catalog-url</ows:Identifier>
<wps:Data>
<wps:LiteralData>dods://hydro1.sci.gsfc.nasa.gov:80/dods/NLDAS_FORA0125_H.002</wps:LiteralData>
</wps:Data>
</wps:Input>
<wps:Input>
<ows:Identifier>allow-cached-response</ows:Identifier>
<wps:Data>
<wps:LiteralData>false</wps:LiteralData>
</wps:Data>
</wps:Input>
</wps:DataInputs>
<wps:ResponseForm>
<wps:RawDataOutput mimeType="application/json">
<ows:Identifier>result_as_json</ows:Identifier>
</wps:RawDataOutput>
</wps:ResponseForm>
</wps:Execute>

**Comment by jread-usgs** [comment]
This is fixed in #57 for opendap, but it is xml for wcs (and entirely different). uggg wcs.

**Comment by jread-usgs** [comment]
closing this for opendap, creating a wontfix for WCS (for the present time)
